web/docs/data-model/operations/actions.md

```diff
@@ -545,21 +533,23 @@ Declaring the Action:
 
 ```wasp
 query createFoo {
-    fn: import { createFoo } from "@server/actions.js"
+    fn: import { createFoo } from "@src/actions.js"
     entities: [Foo]
 }
 ```
 
-And also creates a type you can import on the server:
+Enables you to import and use it anywhere in your code (on the server or the client):
 
 ```ts
-import createFoo from '@wasp/actions'
+import { createFoo } from 'wasp/client/operations'
+// TODO(miho): missing API
```

Comment:
Ok, we need to figure this out indeed. I wonder if this ever worked on server, I guess it did, but I am not sure.